src/main/java/redis/clients/jedis/HighLevelBinaryJedisClusterCommands.java

```diff
@@ -0,0 +1,5 @@
+package redis.clients.jedis;
+
+public interface HighLevelBinaryJedisClusterCommands {
```

Comment:
@marcosnils Well , it does a bit more than just sending the commands to the cluster as the other implmentations do . I also didn't want to interfere with the other  interface . Let me know what you decide and I'll change it  according to your request